### PR TITLE
upgrade to spring 2.2.6 and tracer 2.0.0

### PR DESCRIPTION
**Pull request by gopesh67**

### CODE REVIEW

File: egov-url-shortening/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+
+# Changelog
+All notable changes to this module will be documented in this file.
+
+## 1.1.1 - 2020-06-12
```
Comment (tarunlalwani) [inline egov-url-shortening/CHANGELOG.md:6]: Pom says `1.0.0` and your change log says `1.1.1`?

File: egov-url-shortening/pom.xml [hunk 2]
```diff
@@ -50,7 +72,11 @@
 		<dependency>
 			<groupId>org.egov.services</groupId>
 			<artifactId>tracer</artifactId>
-			<version>1.1.5-SNAPSHOT</version>
+			<version>2.0.0-SNAPSHOT</version>
```
Comment (tarunlalwani) [inline egov-url-shortening/pom.xml:75]: Any reason we added redis?

File: egov-url-shortening/src/main/resources/application.properties [hunk 3]
```diff
@@ -12,14 +14,14 @@ spring.datasource.username=postgres
...
-flyway.password=postgres
-flyway.table=public
-flyway.baseline-on-migrate=true
-flyway.outOfOrder=true
-flyway.locations=db/migration/main
-flyway.enabled=true
+spring.flyway.url=jdbc:postgresql://localhost:5432/chat
+spring.flyway.user=postgres
+spring.flyway.password=postgres
+spring.flyway.table=public
```
Comment (tarunlalwani) [inline egov-url-shortening/src/main/resources/application.properties:20]: Why are you changing db ?

File: egov-url-shortening/pom.xml (hunk 2 above)
Comment (gopesh67) [reply egov-url-shortening/pom.xml:75]: In spring boot newer version Lettuce is used as default connector, to use Jedis connector we need to add dependency

**Pull request merged**